inc/reminder.class.php

```diff
@@ -898,14 +919,24 @@ static function showListForCentral($personal = true) {
          $rand = mt_rand();
 
          while ($data = $iterator->next()) {
+            $self = new self();
+            $self->getFromDB($data["id"]);
```

Comment:
Unless I miss something, this `$self` object is not used.

Reply:
Corrected thx